collection-spec/collection-spec.md

```diff
@@ -146,107 +145,19 @@ Implementors are free to add other derived statistical values to the object, for
-  "title": "Landsat 8 L1",
-  "description": "Landat 8 imagery radiometrically calibrated and orthorectified using gound points and Digital Elevation Model (DEM) data to correct relief displacement.",
-  "extent": {...},
-  "license": "PDDL-1.0",
-  "properties": {
-    "eo:gsd": 30,
-    "platform": "landsat-8",
-    "instruments": ["oli", "tirs"],
-    "eo:off_nadir": 0,
-    "eo:bands": [
-      {
-        "name": "B1",
-        "common_name": "coastal",
-        "gsd": 30,
-        "center_wavelength": 0.44,
-        "full_width_half_max": 0.02
-      },
-      ...
-    ]
-  },
-  "links": [...]
-}
-```
-
-An incomplete item:
-```
-{
-  "stac_version": "0.8.1",
-  "type": "Feature",
-  "id": "LC08_L1TP_107018_20181001_20181001_01_RT",
-  "bbox": [...],
-  "geometry": {...},
-  "properties": {
-    "collection": "landsat-8-l1",
-    "datetime": "2018-10-01T01:08:32.033Z",
-    "eo:cloud_cover": 78,
-    "eo:sun_azimuth": 168.8989761,
-    "eo:sun_elevation": 26.32596431,
-    "landsat:path": 107,
-    "landsat:row": 18
-  },
-  "assets": {...},
-  "links": [...]
-}
-```
-
-The merged Item then looks like this:
-
-```
-{
-  "stac_version": "0.8.1",
-  "type": "Feature",
-  "id": "LC08_L1TP_107018_20181001_20181001_01_RT",
-  "bbox": [],
-  "geometry": {},
-  "properties": {
-    "collection": "landsat-8-l1",
-    "datetime": "2018-10-01T01:08:32.033Z",
-    "eo:cloud_cover": 78,
-    "eo:sun_azimuth": 168.8989761,
-    "eo:sun_elevation": 26.32596431,
-    "landsat:path": 107,
-    "landsat:row": 18,
-    "eo:gsd": 30,
-    "platform": "landsat-8",
-    "constellation": "landsat-8",
-    "instruments": ["oli", "tirs"],
-    "eo:off_nadir": 0,
-    "eo:bands": [
-      {
-        "name": "B1",
-        "common_name": "coastal",
-        "gsd": 30,
-        "center_wavelength": 0.44,
-        "full_width_half_max": 0.02
-      },
-      ...
-    ]
-  },
-  "assets": {...},
-  "links": [...]
-}
-```
+## Standalone Collections
+
+STAC Collections which don't link to any Item are called **standalone Collections**.
+To describe them with more fields than the Collection fields has to offer, it is allowed to re-use the metadata fields defined by content extensions for Items in the `summaries` field.
+This makes much sense for fields such as `platform` or `eo:epsg`, which are often the same for a whole collection, but doesn't make much sense for `eo:cloud_cover`, which usually varies heavily across a Collection.
+The data provider is free to decide, which fields are reasoable to be used.
 
 ## Extensions
 
+Important related extensions for the STAC Collection Specification:
+
+* [Commons extension](../extensions/commons/README.md), which primarily allows to add shared Item metadata to Collections,
+  but could also be used to describe Collections better that are not referring to Items by adding additional fields from Item extensions.
+  Please note that this extension is only in '[proposal](../extensions/README.md#extension-maturity)' stage.
```

Comment:
Can I interpret this 'which primarily allows to add shared Item metadata to Collections,
  but could also be used to describe Collections better that are not referring to Items by adding additional fields from Item extensions.'

As this
`which can be used for fields with shared values for all Items in a Collection. Core, extension, and supplemental user-defined Item properties can be defined using this extension.`?

As in: this extension allows users to add wild-card fields at the collection level to describe the items in the collection that do not require validation but support a better understanding of the collection contents?

Reply:
Sorry, this PR was merged without me having a chance to answer. Will answer tomorrow and potentially issue a new PR, if I think there's room for a better description.

Reply:
> which primarily allows to add shared Item metadata to Collections,

This part is still valid.

> but could also be used to describe Collections better that are not referring to Items by adding additional fields from Item extensions.

This part is not. It was used for additional fields before we introduced summaries. Therefore I'll remove this part and issue a new PR.